react/ActionMenu/Readme.md

```diff
@@ -2,224 +2,105 @@ Use an ActionMenu to show a list of actions. ActionMenus automatically switch th
-user is not lost, so the ActionMenuHeader would be redundant. This is
-why it is not rendered unless we are on mobile.
-
-```jsx
-import ActionMenu, { ActionMenuItem, ActionMenuHeader } from './index';
-import DropdownButton from 'cozy-ui/transpiled/react/DropdownButton';
-import Icon from '../Icon';
-import Filename from '../Filename';
-import {
-  BreakpointsProvider
-} from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
-import FileIcon from "cozy-ui/transpiled/react/Icons/File";
-import RightIcon from "cozy-ui/transpiled/react/Icons/Right";
-
-initialState = { menuDisplayed: isTesting() };
-
-const showMenu = () => setState({ menuDisplayed: true });
-const hideMenu = () => setState({ menuDisplayed: false });
-
-<BreakpointsProvider>
-  <DropdownButton onClick={showMenu}>Show action menu</DropdownButton>
-  {state.menuDisplayed &&
-    <ActionMenu
-      onClose={hideMenu}>
-      <ActionMenuHeader>
-        <Filename icon={FileIcon} filename="my_awesome_paper" extension=".pdf" />
-      </ActionMenuHeader>
-      <ActionMenuItem left={<Icon icon={FileIcon} />}>Item 1</ActionMenuItem>
-      <ActionMenuItem left={<Icon icon={RightIcon} />}>Item 2</ActionMenuItem>
-      <ActionMenuItem left={<Icon icon={FileIcon} />}>Item 3</ActionMenuItem>
-  </ActionMenu>}
-</BreakpointsProvider>
-```
-
-### With dedicated click handler
-
-```jsx
-import ActionMenu, { ActionMenuItem, ActionMenuHeader } from './index';
-import DropdownButton from 'cozy-ui/transpiled/react/DropdownButton';
-import Icon from '../Icon';
-import Filename from '../Filename';
-import {
-  BreakpointsProvider
-} from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
-import FileIcon from "cozy-ui/transpiled/react/Icons/File";
-
-initialState = { menuDisplayed: isTesting() };
-
-const showMenu = () => setState({ menuDisplayed: true });
-const hideMenu = () => setState({ menuDisplayed: false });
-
-<BreakpointsProvider>
-  <DropdownButton onClick={showMenu}>Show action menu</DropdownButton>
-  {state.menuDisplayed &&
-    <ActionMenu
-      onClose={hideMenu}>
-      <ActionMenuHeader>
-        <Filename icon={FileIcon} filename="my_awesome_paper" extension=".pdf" />
-      </ActionMenuHeader>
-      <ActionMenuItem onClick={() => alert('click')}left={<Icon icon={FileIcon} />}>Item 1</ActionMenuItem>
-  </ActionMenu>}
-</BreakpointsProvider>
-```
-
-### Placement on desktop
+A header can be used to provide context on the menu actions. Since on desktop, we display a popper and not a BottomDrawer, context for the user is not lost, so the ActionMenuHeader would be redundant. This is why it is not rendered unless we are on mobile.
 
 We use [popper.js](https://popper.js.org/docs/v2/) under the hood. You can use the `popperOptions` prop to pass options to the popper.js instance. This lets you control things like placement relative to the anchor, positioning strategies and more — refer to the popper doc for all the details.
```

Comment:
By removing the title we loose the info about this option being useful only on desktop mode.

Maybe we should mention it at the end of paragraph as for the header displayed only on mobile?
Or maybe we can change the the variant name for `withDesktopPlacement` (in opposition to `withMobileHeader`)?
Or both 😄